frontend/packages/modelServing/modelRegistry/DeployRegisteredVersionModal.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import DeployPrefilledModelModal from '@odh-dashboard/internal/pages/modelServing/screens/projects/DeployPrefilledModelModal';
+import { ModelDeployPrefillInfo } from '@odh-dashboard/internal/pages/modelServing/screens/projects/usePrefillModelDeployModal';
+
+export type DeployRegisteredVersionModalProps = {
+  data: {
+    modelDeployPrefillInfo: ModelDeployPrefillInfo;
+    loaded: boolean;
+    error: Error | undefined;
+    onSubmit: () => void;
+  };
+  onClose: () => void;
+};
+
+export const DeployRegisteredVersionModal: React.FC<DeployRegisteredVersionModalProps> = ({
+  data,
+  onClose,
+}) => {
+  const { modelDeployPrefillInfo, loaded, error, onSubmit } = data;
+
+  return (
+    <DeployPrefilledModelModal
+      modelDeployPrefillInfo={modelDeployPrefillInfo}
+      prefillInfoLoaded={loaded}
+      prefillInfoLoadError={error}
+      projectLinkExtraUrlParams={{
+        modelRegistryName: modelDeployPrefillInfo.modelRegistryInfo?.mrName,
+        registeredModelId: modelDeployPrefillInfo.modelRegistryInfo?.registeredModelId,
+        modelVersionId: modelDeployPrefillInfo.modelRegistryInfo?.modelVersionId,
+      }}
```

Comment:
Why does `DeployPrefilledModelModal` accept both a value of `modelDeployPrefillInfo` and extracted values into `projectLinkExtraUrlParams`. Can't the component pull the values itself and just accept the one info prop?

Reply:
Hmm, it's a dashboard component that's not migrated yet, I will check it and refactor that component as needed.